testkit/src/main/java/org/apache/calcite/test/QuidemTest.java

```diff
@@ -293,6 +293,7 @@ public Connection connect(String name) throws Exception {
             .connect();
       case "catchall":
         return CalciteAssert.that()
+            .with(CalciteConnectionProperty.TIME_ZONE, "UTC")
```

Comment:
I am wondering if this impacts our CI which tests under different timezones (see [this](https://github.com/apache/calcite/blob/687dec0afcfab781f905b16e422bc03bd6b9209e/.github/workflows/main.yml#L142-L165) for instance).

Can you try locally to run with different timezones and see if those tests are affected?

You can use ```./gradlew assemble --no-build-cache cleanTest $yourTestsHere -Duser.timezone=$TZ``` for that.

Reply:
I made a change in DateTimeUtils (see CALCITE-2989) that ensures the date and time accessors in AbstractCursor have the same behavior for their getString() methods, and that required a change here to fix CoreQuidemTest. The default time zone is the local time zone, but CatchallSchema is creating `java.sql.Date`, `java.sql.Time`, and `java.sql.Timestamp` values using the UTC time zone. I verified the tests are passing in multiple timezones: UTC, America/Los_Angeles, America/New_York, Pacific/Chatham, Pacific/Samoa, and Pacific/Tongatapu.